hubble/executor/hubio.py

```diff
@@ -840,24 +840,25 @@ def _prettyprint_status_usage(
         )
 
         param_str.add_column(
-            'If you don\'t want to wait for the Executor\'s build status, you can use \'Ctrl + C\'. '
-            'After that you can get the status of the Executor any time using the command.'
+            'If you don\'t want to wait for the build result, feel free to interrupt with \'C-C\'. '
```

Comment:
Can `C-C` be understood by users? Or just a typo here.

Reply:
On PC it's `Ctrl` but on Mac probably `Command`.